pandas/core/internals/managers.py

```diff
@@ -432,10 +418,8 @@ def apply(
             result_blocks = _extend_blocks(applied, result_blocks)
 
         if len(result_blocks) == 0:
-            return self.make_empty(axes or self.axes)
-        bm = self.__class__(
-            result_blocks, axes or self.axes, do_integrity_check=do_integrity_check
-        )
+            return self.make_empty(self.axes)
+        bm = self.__class__(result_blocks, self.axes, do_integrity_check=False)
```

Comment:
The original had ``axes or self.axes``. could we not risk that an ``axes``parameter is in kwargs?

Would it be possible ti be explicit about what parameter thid accepts, i.e. not use kwargs?

Reply:
> The original had axes or self.axes. could we not risk that an axesparameter is in kwargs?

The only case that passed axes was copy, which this changes to not do.

> Would it be possible ti be explicit about what parameter this method accepts, i.e. not use kwargs?

I'm hopeful we'll get there before long